static/js/read_receipts.js

```diff
@@ -0,0 +1,54 @@
+import $ from "jquery";
+import SimpleBar from "simplebar";
+
+import render_read_receipts from "../templates/read_receipts.hbs";
+import render_read_receipts_modal from "../templates/read_receipts_modal.hbs";
+
+import * as channel from "./channel";
+import {$t} from "./i18n";
+import * as loading from "./loading";
+import * as message_store from "./message_store";
+import * as overlays from "./overlays";
+import * as people from "./people";
+
+export function show_user_list(message_id) {
+    $("body").append(render_read_receipts_modal());
+    overlays.open_modal("read_receipts_modal", {
+        autoremove: true,
+        micromodal: true,
+        on_show: () => {
+            loading.make_indicator($("#read_receipts_modal .loading_indicator"));
+            channel.get({
+                url: `/json/messages/${message_id}/read_receipts`,
+                idempotent: true,
+                success(data) {
+                    const message_author_id = message_store.get(message_id).sender_id;
+                    const user_ids = data.user_ids.filter((id) => id !== message_author_id); // Do not show author.
```

Comment:
Comments should be on their own line. Also, we should exclude the sender too, right?

Reply:
We were excluding the sender... Maybe you meant current user? But we don't need to exclude either one -- sender is excluded in the API; we [want](https://chat.zulip.org/#narrow/stream/378-api-design/topic/read.20receipts/near/1365286) to show current user.

Reply:
Yeah, we've settled on just excluding the sender in the API, and nothing here, thanks for confirming.